### ISSUE TITLE
Screenshot gets uploaded to fast

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#403]
It seems that after taking a screenshot the image is post-processed, so that an immediately upload does not transfer the complete file.
My "solution" would be to just wait 1-2s. (It only occurs if you allow the upload to start immediately).

@AndyScherzinger

**Comment by AndyScherzinger** [comment]
Sound perfect to me and should probably also be like that in general since the Google Photos camera app might also do HDR post processing, so waiting for some seconds might be a good thing in general.

**Comment by aronovgj** [comment]
https://github.com/nextcloud/android/issues/85#issuecomment-261759202

Seems related

**Comment by Ventusfahrer** [comment]
Isn't this one a duplicate of #399?

**Comment by AndyScherzinger** [comment]
@Ventusfahrer might be. @tobiasKaminsky what do you think? It seems to be a duplicate of #399.

**Comment by AndyScherzinger** [comment]
This issue in part will probably solve #399, #390 and #397

**Comment by tobiasKaminsky** [comment]
We "now" wait at least 10s before starting auto upload.
